crates/uv/tests/edit.rs

```diff
@@ -374,6 +374,128 @@ fn add_unnamed() -> Result<()> {
     Ok(())
 }
 
+/// Add a development dependency.
+#[test]
+fn add_dev() -> Result<()> {
+    let context = TestContext::new("3.12");
+
+    let pyproject_toml = context.temp_dir.child("pyproject.toml");
+    pyproject_toml.write_str(indoc! {r#"
+        [project]
+        name = "project"
+        version = "0.1.0"
+        requires-python = ">=3.12"
+        dependencies = []
+    "#})?;
+
+    uv_snapshot!(context.filters(), context.add(&["anyio==3.7.0"], true), @r###"
```

Comment:
Similar to above, why are we toggling the Windows filters?

Reply:
I'm a little confused, where's the `--dev` flag?

Reply:
Ah okay I see now that you're using `true`/`false` to toggle dev not the `windows_filters`.

Reply:
See https://github.com/astral-sh/uv/pull/4327#discussion_r1640130152